.github/workflows/workflow-tester.yml

```diff
@@ -11,35 +11,132 @@
 name: Workflow Tester
```

Comment:
When you want to make a new workflow, you have no way to test it without checking it in once. So I made a scratch workflow (`workflow-tester`) that is only run with workflow-dispatch.

In this case, I didn't need to do it, because just making the PR ran all of the tests, but I had already cut'n'pasted it into the workflow-tester and didn't undo it.